dnf-behave-tests/dnf/installonlypkgs.feature

```diff
@@ -59,3 +59,17 @@ Scenario: Clear the list of installonly packages and set the package installonly
         | install       | installonlyA-0:2.0-1.x86_64           |
         | upgrade       | installonlyB-0:2.0-1.x86_64           |
         | upgrade       | kernel-core-0:4.20.6-300.fc29.x86_64  |
+
+
+Scenario: Upgrade doesn't install older installonly pkg when never version is already installed
```

Comment:
Sorry about that, I also noticed that only today.
I made a PR which backports the fix to dnf5: https://github.com/rpm-software-management/libdnf/pull/1564 but it will need review so maybe it would be more convenient to disable it in the meantime.

Reply:
I've merged the patch :)